cli/src/command/wallet.rs

```diff
@@ -197,6 +197,8 @@ pub async fn new_command(
 }
 
 pub async fn restore_command(storage_path: &Path, snapshot_path: &Path, backup_path: &Path) -> Result<Wallet, Error> {
+    check_file_exits(backup_path).await?;
```

Comment:
https://doc.rust-lang.org/std/path/struct.Path.html#method.exists

Reply:
Ah sorry I meant to send the tokio version

Reply:
And I wasn't aware of the permission thing

Reply:
ok I used `fs::try_exists` from tokio, but still this is in a separate fucntion, because it requires error conversion for two cases.